api/src/services/attachment-service.ts

```diff
@@ -33,6 +35,19 @@ export class AttachmentService extends DBService {
     this.attachmentRepository = new AttachmentRepository(connection);
   }
 
+  async testSubmitAttachments(projectId: number) {
+    const platformService = new PlatformService(this.connection);
+    const dataPackageId = v4();
+    const attachmentIds = (await this.getProjectAttachments(projectId)).map((attachment) => attachment.id);
+    const reportAttachmentIds = (await this.getProjectReportAttachments(projectId)).map((attachment) => attachment.id);
+    await platformService.uploadProjectAttachmentsToBioHub(
+      dataPackageId,
+      projectId,
+      attachmentIds,
+      reportAttachmentIds
+    );
+  }
+
```

Comment:
We should probably remove this before merging. Good idea on how to test it, and could be used locally to demo it is working, but lets not merge it to dev. We'll be starting on the UIs to trigger this next sprint, so it won't be longer before we can run it for real.